dipy/reconst/shm.py

```diff
@@ -231,6 +231,60 @@ def real_sph_harm(m, n, theta, phi):
 
 
 def real_sym_sh_mrtrix(sh_order, theta, phi):
```

Comment:
This is the symmetric version. This needed change too? Will changing this affect other areas of the existing code?

Reply:
Hi. Git failed to diff properly, but the real change is the addition of the function `real_full_sh_mrtrix`. The function `real_sym_sh_mrtrix` is the same as before and therefore these changes shouldn't affect other areas of the existing code.